common/src/main/java/com/wynntils/models/players/GuildModel.java

```diff
@@ -119,6 +141,19 @@ public void onChatMessage(ChatMessageReceivedEvent e) {
             guildRank = GuildRank.valueOf(rankChangedMatcher.group(2).toUpperCase(Locale.ROOT));
             WynntilsMod.info("User's guild rank changed to " + guildRank);
         }
+
+        Matcher objectiveCompletedMatcher = message.getMatcher(MSG_OBJECTIVE_COMPLETED);
+        if (objectiveCompletedMatcher.matches()) {
+            updateObjectivesCompletedProgress(objectivesCompletedProgress.current() + 1);
+            if (objectiveCompletedMatcher.group("player").equals(McUtils.playerName())) {
```

Comment:
I'm 99% sure that it will display the player name, as I logged a message testing the OphanimBar overlay of a nicked GuildMember completing their objective (which just displayed their username).